tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -166,6 +180,10 @@ def _return_class_from_tap(self, return_class='ContactDetails'):
                 return EditContact(self.marionette)
             elif return_class == 'SelectContact':
                 return None
+            elif return_class == 'SetIceContacts':
```

Comment:
You might want to add a `Contacts(self.marionette).wait_to_be_displayed()` here.